### PR TITLE
Add form elements for interactive qr code generation

### PR DESCRIPTION
**Pull request by tstoppe89**
Added js functionality with fetch API to 
- create and display invoices, 
- send preimage and text data for generating qr code
- display qr code blob as image

### CODE REVIEW

File: www/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ APIs
 
 Generates a QR code for the given data.
 
-1. Generate an invoice: `curl https://api.lightning.ws/qr`
-2. Pay the invoice that's in the response body
-3. Send the request with the preimage as payment proof (hex encoded) and the data you want in the QR code as query parameter: `curl -H "x-preimage: <PAYMENT_PREIMAGE>" https://api.lightning.ws/qr?data=testtext`
-4. The response is the QR code as PNG image
+1. Generate an invoice: `curl https://api.lightning.ws/qr` <div id="generateBtnContainer"><button onclick="generateInvoice()">generate invoice now</button></div>
```
Comment (philippgille) [inline www/index.md]: `Copy to clipboard` is capitalized, `Generate ...` should be the same

File: www/index.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ APIs
...
+2. Pay the invoice that's in the response body <div id="responseContainer"><input type="text" id="responseArea"><button onclick="copyToClipboard('responseArea')">Copy to clipboard</button></div>
+3. Send the request with the preimage as payment proof (hex encoded) and the data you want in the QR code as query parameter: `curl -H "x-preimage: <PAYMENT_PREIMAGE>" https://api.lightning.ws/qr?data=testtext` <div id="preimageContainer"><input id="preimage" placeholder="preimage" type="text" /><textarea placeholder="data" id="dataArea"></textarea><button onclick="sendPreImage()">create qr code</button><div id="sendError"></div></div>
```
Comment (philippgille) [inline www/index.md]: Capitalize `create...`, capitalize `QR`

File: www/js/qrCode.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        return response.text();
+    }).then(function (data){
+        show("responseContainer");
+        show("preimageContainer");
+        setValueById("responseArea", data);
+    });
+}
+
+
+function sendPreImage(){
```
Comment (philippgille) [inline www/js/qrCode.js]: `Preimage`, not `PreImage`

**Review by philippgille — CHANGES_REQUESTED** [review]

File: www/index.md (hunk 2 above)
Comment (tstoppe89) [reply www/index.md]: All Letters are capitalized by default CSS design in buttons

**Pull request merged**